TheHiveV5/action_create_alert.json

```
@@ -14,6 +14,27 @@
       "alert": {
         "description": "A Sekoia.io alert",
         "type": "object"
+      },
+      "tlp": {
+        "description": "Case's TLP: 0, 1, 2, 3, 4",
+        "type": "integer",
+        "enum": [
+          0,
+          1,
+          2,
+          3,
+          4
+        ]
```

According to your code, this argument must be nullable that is not the case here.
I propose the following change to match the criterion:
```suggestion
        "anyOf": [ 
          {
            "type": "integer",
            "enum": [
              0,
              1,
              2,
              3,
              4
            ]
          },
          {
            "type": "null"
          }
        ],
        "default": null
```

see [Sekoia.io action](https://github.com/SEKOIA-IO/automation-library/blob/main/Sekoia.io/action_deletes_an_asset_v2.json#L30-L40)